### ISSUE TITLE
SAF-T is an international standard for electronic exchange of reliable accounting data from organizations to a national tax authority or external auditors.

### ISSUE THREAD

**Issue by RTEIS31** [frappe/erpnext#37961]
Can Frappe/ERNext implement the option or possibilities for ERPNext to produce XML SAF-T file as required our Customs authority for Taxes checking.

SAF-T: SAF-T is an international standard for electronic exchange of reliable accounting data from organizations to a national tax authority or external auditors. (https://edicomgroup.com/learning-center/what-is-electronic-tax-declaration/what-is-saft-system)"

We can not Sell Frappe/ERPnext solutions in Angola due to not having the possibilities of creating XML type files to produce SAF-T files

**Comment by bosue** [comment]
Yeah, even though it may have only be adopted by a few dozens of countries, this is an international standard, so should definitely be supported out-of-the-box.

We will however need someone from a SAF-T adopting country to lead this feature’s development.

Don’t know about adoption rate and planned legislation, but I guess this could possibly be interesting for the French market, @FHenry?

**Comment by linofcp007** [comment]
any news on this?
